tests/core/test_processor.py

```diff
@@ -356,9 +356,7 @@ def default_reminder_list():
     ]
 
 
-async def test_reminder_cancelled_by_name(
-    default_channel: CollectingOutputChannel, default_processor: MessageProcessor
-):
+async def tracker_with_scheduled_reminder(default_processor: MessageProcessor) -> DialogueStateTracker:
```

Comment:
using a `pytest.fixture` would be bit more elegant imo

Reply:
Ah. I didn't know what the was. Looked it up and changed it now.